UnityProject/Assets/Scripts/Systems/InGameEvents/InGameEventScripts/EventBureaucraticError.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Collections;
+using UnityEngine;
+using System.Linq;
+using Managers;
+using Strings;
+
+namespace InGameEvents
+{
+	public class EventBureaucraticError : EventScriptBase
+	{
+		public override void OnEventStart()
+		{
+			if (AnnounceEvent)
+			{
+				var text = "A recent bureaucratic error in the Organic Resources Department " +
+					"may result in personnel shortages in some departments and redundant staffing in others.";
+
+				CentComm.MakeAnnouncement(ChatTemplates.CentcomAnnounce, text, CentComm.UpdateSound.Alert);
+			}
+
+			if (FakeEvent) return;
+
+			base.OnEventStart();
+		}
+
+		public override void OnEventStartTimed()
+		{
+			OccupationList.Instance.Occupations.Where(o => o.IsCrewmember).PickRandom().SetLimit(-1);
```

Comment:
could also choose another role to reduce the limit, maybe

Reply:
How do you mean?

Reply:
well, descriptions says "shortage on others". Does ss13 just give a single role unlimited slots?

Reply:
Oh I getcha. https://tgstation13.org/wiki/Random_events#Bureaucratic_Error seems to imply one role. I can change it, or the text, if you think it'll be better.

Reply:
The text also seems to imply that there can be one "overflow role", though I've seen situations where there have been more than one.

Reply:
uhhhm, wiki says it shifts the unlimited role so you should give assistants a limit.